azurelinuxagent/common/osutil/default.py

```diff
@@ -95,6 +95,9 @@
 
 IP_COMMAND_OUTPUT = re.compile('^\d+:\s+(\w+):\s+(.*)$')
 
+BASE_CGROUPS = '/sys/fs/cgroup'
+
+
 class DefaultOSUtil(object):
```

Comment:
If you bring this into DefaultOSUtil I would have expected more edits where you would not implement it, such as FreeBSD, etc.

Reply:
This will fail gracefully on BSD or anywhere which does not have the base path or where cgroups cannot be mounted.

Reply:
SGTM.